src/Markdig/MarkdownPipelineBuilder.cs

```diff
@@ -91,6 +91,17 @@ public MarkdownPipelineBuilder()
         internal ProcessDocumentDelegate GetDocumentProcessed => DocumentProcessed;
 
         /// <summary>
+        /// Adds the specified extension to the extensions collection.
+        /// </summary>
+        /// <typeparam name="TExtension">The type of the extension.</typeparam>
+        /// <returns>The current instance of <see cref="MarkdownPipeline" /></returns>
+        public MarkdownPipelineBuilder Use<TExtension>() where TExtension : class, IMarkdownExtension, new()
```

Comment:
Couldn't we move this as an extension method to MarkdownExtension to keep it close to other explicit Use?